frog/frog.rkt

```diff
@@ -351,7 +357,9 @@
   (clean-post-output-files)
   (clean-non-post-output-files)
   (clean-tag-output-files)
-  (clean-serialized-posts))
+  (clean-serialized-posts)
+  (when (current-responsive-images?)
```

Comment:
Maybe `clean-resized-images` should run unconditionally?  e.g. Someone decides to stop using it, updates their config file, then runs `raco frog --clean` -- they'd expect it to clean.